tests/phpunit/admin/class-wc-stripe-whats-new-modal-test.php

```
@@ -0,0 +1,225 @@
+	public function test_standalone_update_with_single_plugin_sets_transient(): void {
+		$this->modal->maybe_flag_standalone_update(
+			null,
+			[
+				'action'  => 'update',
+				'type'    => 'plugin',
+				'plugins' => [ plugin_basename( WC_STRIPE_MAIN_FILE ) ],
+			]
+		);
+
+		$this->assertSame( '1', get_transient( WC_Stripe_Whats_New_Modal::PENDING_TRANSIENT ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_standalone_update_with_single_plugin_sets_transient is not named in camelCase. (undefined)

(CamelCaseMethodName)